crates/primitives/src/revm/config.rs

```diff
@@ -11,13 +13,13 @@ pub fn revm_spec_by_timestamp_after_merge(
 ) -> revm_primitives::SpecId {
```

Comment:
once merged we can remove this from primitives and have two separate functions for this